libavpipe/src/avpipe_xc.c

```diff
@@ -3323,6 +3329,23 @@ get_filter_str(
 {
     *filter_str = NULL;
 
+    if (params->rotate > 0) {
```

Comment:
As is, this breaks if used in combination with watermarking, right? It also seems to avoid existing applications of `scale`. It seems to make more sense for this function to successively apply filters than to apply only a single given filter.

Reply:
if rotate is set, it will return the filter string for rotation (one filter string is applied only). If you have WM and rotation, the rotation is applied.

Reply:
Is scale ever used at the same time as rotation?